word_guess.rb

```diff
@@ -0,0 +1,177 @@
+require 'faker'
+
+#creates array with 10 random names. partitions first name of each character
+simpsons = []
+10.times do
+  name = Faker::Simpsons.character.downcase
+  name = name.partition(" ").first
+  simpsons << name
+end
+
+class Game
+  attr_accessor :lives, :word_bank, :initial_tries, :dashes, :flower_pot, :random_word
+
+  def initialize(word_bank)
+    @word_bank = word_bank
+    @random_word = @word_bank.sample
+    @lives = 5
+    @dounut_pic = "
+   _..------.._
+  .'   .-\"\"-.   '.
+  |\\   '----'   /|
+  \\ `'--------'` /
+  '._        _.'
+     `\"\"\"\"\"\"`
+     "
+    @initial_tries = [@dounut_pic, @dounut_pic, @dounut_pic, @dounut_pic, @dounut_pic]
+    @dashes = get_init_dashes
+  end
+
+  # Gives us array of dashes based on number of letters in random_word
+  def get_init_dashes
```

Comment:
Why does the game generate the list of dashes, wouldn't the word be able to do that?